### PR TITLE
Add initial container and clear button

### PR DESCRIPTION
**Pull request by yoksanherlie**

### CODE REVIEW

**Comment by yoksanherlie** [comment]
#7 Done

**Comment by anshumanv** [comment]
Works amazingly well !
This is such an awesome PR :smile_cat:

File: src/control.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+window.addEventListener('load', function() {
+    const btnStart = document.querySelector("#start");
+
+    btnStart.addEventListener("click", () => {
+        document.querySelector(".initial-container").classList.add("hide");
+    });
+});
```
Comment (anshumanv) [inline src/control.js]: Can you perhaps add a newline at the end ?

Comment (yoksanherlie) [reply src/control.js]: At which end?
The btnStart scope or the latest scope?

Comment (anshumanv) [reply src/control.js]: in file `control.js` there is no blank line at the end.
*btw I'm an anime fan too, have seen detective conan* :smile:

**Review by anshumanv — APPROVED** [review]
Thanks !

File: src/control.js (hunk 1 above)
Comment (yoksanherlie) [reply src/control.js]: Done! :smile:

<i>Detective Conan :+1: </i>

Comment (anshumanv) [reply src/control.js]: Perhaps could you send another PR and remove points from Todo in README.md which are covered in this PR ? :smile: 
Do you wish to become a collaborator ?

Comment (yoksanherlie) [reply src/control.js]: I'll do the PR again.

Would really love to become a collaborator! :smile:

Comment (anshumanv) [reply src/control.js]: Invite sent, check mail please :smile_cat:

Comment (yoksanherlie) [reply src/control.js]: Accepted :smile:

**Pull request merged**